---
apiVersion: "v1"
kind: "Event"
metadata:
  labels:
    EntandoResourceKind: "EntandoApp"
    EntandoResourceNamespace: "my-namespace"
    EntandoApp: "my-app"
  name: "my-app-48016090"
  namespace: "my-namespace"
  ownerReferences:
  - apiVersion: "entando.org/v1"
    kind: "EntandoApp"
    blockOwnerDeletion: true
    controller: true
    name: "my-app"
    uid: "1592799a-9605-4e79-a09f-3541e4a50615"
action: "FAILED"
count: 1
firstTimestamp: "2021-06-07T14:35:24Z"
involvedObject:
  apiVersion: "entando.org/v1"
  kind: "EntandoApp"
  fieldPath: "status"
  name: "my-app"
  namespace: "my-namespace"
  resourceVersion: "3344567"
  uid: "1592799a-9605-4e79-a09f-3541e4a50615"
lastTimestamp: "2021-06-07T14:35:24Z"
message: "The deployment of EntandoApp my-namespace/my-app failed due to Fake Controller\
  \ Exception. Fix the root cause and then trigger a redeployment by adding the annotation\
  \ 'entando.org/processing-instruction: force'"
reason: "Failed"
source:
  component: "entando-k8s-app-controller"
type: "Error"
